website-next/components/projects/projects-page-component.tsx

```diff
@@ -109,7 +109,7 @@ class ProjectCard extends React.Component<ProjectCardProps> {
                 fontSize: 11,
                 display: 'inline-block',
                 wordWrap: 'break-word',
-                marginTop: 8,
+                marginTop: 0,
```

Comment:
nitpicking here, but I'd assume if you don't want a margin _at all_ you could just remove this line altogether?

Reply:
true, adjusting that now